tiledb/sm/c_api/tiledb.h

```diff
@@ -901,14 +901,38 @@ TILEDB_EXPORT int32_t tiledb_array_schema_has_attribute(
  *
  * @param ctx The TileDB context.
  * @param array_schema The array schema.
- * @param out The output.
+ * @param out The output file.
```

Comment:
`FILE *` need not return only to files.
```suggestion
 * @param out The output handle.
```